packages/network-controller/src/rpc-service/rpc-service.test.ts

```diff
@@ -83,15 +84,6 @@ describe('RpcService', () => {
       },
     );
 
-    describe('if making the request throws a "Gateway timeout" error', () => {
```

Comment:
This test was kinda broken from the beginning. It was testing when fetch returned a "Gateway timeout" error, but that's not an error that fetch can throw.

It was working in the past because the RPC service was translating 503/504 fetch responses into "Gateway timeout" errors, inside of the policy execution function. Effectively "Gateway timeout" was an expected case. Now that that's no longer the case, these tests no longer worked.

The case where a "Gateway timeout" error is returned (HTTP 503/504 responses) is tested separately from here still.